app/src/test/java/ua/test/mega/tester/junit/AccountProcessorTest.java

```diff
@@ -0,0 +1,127 @@
+package ua.test.mega.tester.junit;
+
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.security.test.context.support.WithAnonymousUser;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.junit4.SpringRunner;
+import ua.test.mega.tester.adapters.AccountAdapterInMemory;
+import ua.test.mega.tester.adapters.LoggedInUserAdapterForSpringSecurity;
+import ua.test.mega.tester.adapters.UserAdapterInMemory;
+import ua.test.mega.tester.core.AccountProcessor;
+import ua.test.mega.tester.core.api.AccountAdapter;
+import ua.test.mega.tester.core.api.LoggedInUserAdapter;
+import ua.test.mega.tester.core.api.UserAdapter;
+import ua.test.mega.tester.core.api.model.Account;
+
+import java.math.BigDecimal;
+
+
+@RunWith(SpringRunner.class)
+public class AccountProcessorTest {
+    private static Account actualFirstAccount;
+    private static Account actualSecondAccount;
+
+    private static UserAdapter userAdapter;
+    private AccountProcessor accountProcessor;
+
+
+    @BeforeClass
+    public static void init() {
+        userAdapter = new UserAdapterInMemory();
+
+        actualFirstAccount = Account.builder()
+                .accountId(1L)
+                .balanceInUSD(new BigDecimal(1000))
+                .build();
+
+        actualSecondAccount = Account.builder()
+                .accountId(-1L)
+                .balanceInUSD(new BigDecimal(1000))
+                .build();
+    }
+
+    @Before
+    public void setUp() {
+        LoggedInUserAdapter loggedInUserAdapter = new LoggedInUserAdapterForSpringSecurity(userAdapter);
+        AccountAdapter accountAdapter = new AccountAdapterInMemory(userAdapter);
+
+        accountProcessor = new AccountProcessor(loggedInUserAdapter, accountAdapter);
```

Comment:
Is it possible to create this object with spring?

Reply:
yep